src/styles/utiles/_reset.scss

```diff
@@ -0,0 +1,8 @@
+body,
+p,
+h1,
+h2,
+h3,
+ul {
+  margin: 0;
+}
```

Comment:
if i will use reset class to each tag that i use it will be very complicated to find where i use this class where not. 
i decided to reset all in one place.

Reply:
I understand, but why than you added a class='reset' to HTML that you don't use on CSS

Reply:
oh yep i got it thanks)